_data/open.yml

```diff
@@ -8217,7 +8217,7 @@ concepts:
       property: fenced-stacked
       area: "combinatorics"
       comments:
-       - "<math><mrow intent='multichoose($a1,$a2)'><mo mathvariant='bold'>(</mo><mfrac linethickness='0pt'><mi arg='a1'>n</mi><mi arg='a2'>n</mi></mfrac><mo mathvariant='bold'>)</mo></mrow></math>"
+       - "<math><mrow intent='multichoose($a1,$a2)'><mo mathvariant='bold'>(</mo><mfrac linethickness='0pt'><mi arg='a1'>m</mi><mi arg='a2'>n</mi></mfrac><mo mathvariant='bold'>)</mo></mrow></math>"
```

Comment:
I see that a lot of lines were changed. However this is the only line that causes a warning in Firefox. Thus I was thinking it would be good to come from 1 to 0 warning. I have no particular feeling about the unicode character.

Reply:
@physikerwelt  ah sorry yes the warning is the mathvariant bold which isn't actually valid i added it as unicode doesn't have bold brackets I;'ll replace bycss. Firefox also tells me Invalid markup: Incorrect number of children for <msup/> tag.`  update coming up....

Reply:
@physikerwelt  thanks, fixed